### DISCUSSION TITLE
Special feature request

### DISCUSSION THREAD

**Discussion by JeffroDB** [samemory/homebridge-eufy-security-control#244] [Ideas]
I'm currently using three EufyCam 2 Pro's and have enabled the native iOS Eufy app's /Homekit integration for all three. The cameras are successfully showing up in Apple Home, recording video to my Apple iCloud account and I've been able to utilize their motion sensors to create outdoor security lighting automations with other accessories within Apple Home.

The one feature in home bridge-eufy-security that I truly miss since I uninstalled it from my Mac-based Homekit configuration is the accessory it created and exposed to Homekit that allowed me (via either the iOS UI or a Siri command spoken to my Homepods) to change the security "mode" of the native Eufy security app, i.e. to "Home", "Away", "Disarmed".

Is there any way you could modify this app's optional settings so that it's the only accessory created?

**Comment by lenoxys** [comment]
Hello,

When doing the wizard at the first step you can ignore device just ignore all of them except station device

**Reply by JeffroDB** [reply]
Done. Thank you very much!

**Comment by ben-connolly** [comment]
Hi. Just installed this plugin hoping it would do for my new Eufy cameras what the Ring plug-in does… pop up the video feed on my Apple TV when motion is detected. The Ring plug-in allows you to configure motion detection to appear as a doorbell press in HomeKit. 

I couldn’t find anyone else asking for this in the issues. Not sure if it’s been requested before?

**Comment by JeffroDB** [comment]
Good morning Ben,

Independent of the plug-in, if the cams are visible to Homekit their motion
detectors are exposed as accessories and usable in automations. For
instance, in my setup I have a cam associated with a “room” which is
actually my back yard. Also in that back yard  “room” is an Apple Home
managed floodlight. I have a simple automation that uses the cam’s motion
detector to light the floods when it senses motion at night. You’ll find
each motion sensor in each cam’s assigned room and a long press exposes its
gear icon and the opportunity to automate.

That was the easy part. :-)

I haven’t a clue how you’d use that motion trigger to start up the AppleTV
to display it’s associated cam’s live feed. Maybe you could ask around the
Ring sub and see how that plugin manages to manage your Apple TV in that
way.

On Sat, Oct 2, 2021 at 7:12 AM Ben Connolly ***@***.***>
wrote:

> Hi. Just installed this plugin hoping it would do for my new Eufy cameras
> what the Ring plug-in does… pop up the video feed on my Apple TV when
> motion is detected. The Ring plug-in allows you to configure motion
> detection to appear as a doorbell press in HomeKit.
>
> I couldn’t find anyone else asking for this in the issues. Not sure if
> it’s been requested before?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/samemory/homebridge-eufy-security/discussions/244#discussioncomment-1418109>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASIQWIG7EYFVJBGMXMWKOVDUE3SLLANCNFSM5EY6476Q>
> .
>
-- 
—
All the best,

Jeffrey

**Comment by JeffroDB** [comment]
Hey Ben,

I mistakenly assumed your email came from Reddit. Only later did I see it
was via Github so that last bit of advice about checking the “sub” is
probably a blind lead. Sorry about that.

I’ve tried and failed to automate the family’s Apple TV many times. There’s
just so little to work with. There is a Homebridge plugin called “Apple TV
Remote” that’s easy enough to get up and running and provides some useful
automation tools. In experimenting with it this morning to try and open our
Apple TV’s EufyCam live stream I failed. Once you open the Apple TV’s
“control center” (that UI that slides out from the right allowing account
changes, sleep mode, Apple Home, etc.) it becomes a black box as far as the
Homebridge log is concerned.



On Sat, Oct 2, 2021 at 7:12 AM Ben Connolly ***@***.***>
wrote:

> Hi. Just installed this plugin hoping it would do for my new Eufy cameras
> what the Ring plug-in does… pop up the video feed on my Apple TV when
> motion is detected. The Ring plug-in allows you to configure motion
> detection to appear as a doorbell press in HomeKit.
>
> I couldn’t find anyone else asking for this in the issues. Not sure if
> it’s been requested before?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/samemory/homebridge-eufy-security/discussions/244#discussioncomment-1418109>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASIQWIG7EYFVJBGMXMWKOVDUE3SLLANCNFSM5EY6476Q>
> .
>
-- 
—
All the best,

Jeffrey

**Reply by ben-connolly** [reply]
Hi Jeffrey - thanks for your reply. Wondering if this helps?

line 105:  

https://github.com/dgreif/ring/blob/master/homebridge/camera.ts

**Reply by ben-connolly** [reply]
Sorry, probably should have addressed that to @lenoxys

**Reply by ben-connolly** [reply]
It looks as though TVOS 15 may have solved this already. Now to test…

https://homekitnews.com/2021/09/25/how-to-add-video-camera-notifications-to-your-apple-tv/

**Comment by JeffroDB** [comment]
Good Lord, I hadn’t upgraded the Apple TV to 15. The Homepods, yes. 😬

Upgrading now. Thank you for the heads-up!



On Sat, Oct 2, 2021 at 4:34 PM Ben Connolly ***@***.***>
wrote:

> It looks as though TVOS 15 may have solved this already. Now to test…
>
>
> https://homekitnews.com/2021/09/25/how-to-add-video-camera-notifications-to-your-apple-tv/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/samemory/homebridge-eufy-security/discussions/244#discussioncomment-1419446>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASIQWICZ67AF3NE3VREQMYDUE5UENANCNFSM5EY6476Q>
> .
>
-- 
—
All the best,

Jeffrey
